utility/src/main/java/org/oppia/util/firebase/CrashLogger.kt

```diff
@@ -0,0 +1,11 @@
+package org.oppia.util.firebase
+
+import java.lang.Exception
+
+/**
+ * Interface that enables exception logging.
+ */
+interface CrashLogger {
+
+  fun logException(exception: Exception)
```

Comment:
Need a java doc comment for this function

Reply:
Done.